performance-storage-service/pss_project/settings/production.py

```diff
@@ -2,3 +2,14 @@
 
 DEBUG = False
 ALLOWED_HOSTS = []
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': get_environ_value('PSS_DATABASE_NAME'),
+        'USER': get_environ_value('PSS_DATABASE_USER'),
+        'PASSWORD': get_environ_value('PSS_DATABASE_PASSWORD'),
+        'HOST': '127.0.0.1',
```

Comment:
`127.0.0.1` refers to the docker itself. No db service is there. Should use service name.